Routines/RoutineBase.cs

```diff
@@ -559,7 +560,9 @@ public bool TryKamakaziPower(TrinityActor target, TrinityPower routinePower, out
             // But for goblin kamakazi we need a special exception to force it to always target the goblin.
 
             power = null;
-            if (!target.IsTreasureGoblin || Core.Settings.Weighting.GoblinPriority != TargetPriority.Kamikaze) return false;
+            if (!target.IsTreasureGoblin
+                || Core.Settings.Weighting.GoblinPriority != TargetPriority.Kamikaze
+                || target.Position == Vector3.Zero) return false;

```

Comment:
Actors with Position of Zero should be excluded in the targeting instead of adding a check here.